server/scriptController.js

```diff
@@ -20,7 +22,102 @@ const scriptController = {
       .catch((error) => {
         return next({
           log: `error: ${error} occured when getting script titles`,
-          message: 'error in getScriptTitles in scriptController' ,
+          message: 'error in getScriptTitles in scriptController',
+        });
+      });
+  },
+  saveScript: (req, res, next) => {
+    const MAXFILESIZE = 50 * 1024 * 1024; //50MB
```

Comment:
Developers often forget to keep constants that exist on both the front end and back end in sync when changing them. Adding a comment on each can help avoid this. EX:
```js
    const MAXFILESIZE = 50 * 1024 * 1024; // 50MB. If updating, change constant in XXX.js too.
```